lib/streamlit/elements/layouts.py

```diff
@@ -93,7 +93,23 @@ def container(
 
         .. output ::
             https://doc-container2.streamlit.app/
-            height: 480px
+            height: 300px
+
+        Using ``height`` to make a grid:
```

Comment:
nit: maybe it would also be nice to add one more example with a single container with a height that actually scrolls (because it contains too many elements)

Reply:
Proposed an example [here](https://github.com/streamlit/docs/pull/921).